scripts/utils/update_package.py

```diff
@@ -104,6 +107,7 @@ def update_github_url(package):
 
     latest_version = None
     for url, org, project, version in matches:
+        # version excludes `v` from the capturing group in the regex therefore latest_version_match mustn't include `v` if the version starts with `v`
```

Comment:
Can you move this comment (maybe with a bit rewording needed) just before the line `if latest_version.startswith('v'):`? Here it is difficult to understand what it is refering to :confused:

Reply:
done in the last commit